poetry/orb.yaml

```diff
@@ -93,6 +93,12 @@ jobs:
           Username to be used for authenticating against the specified
           repository.
         type: string
+      skip_build:
```

Comment:
I'd vote for this being `build` with a default of `true`. I think where possible, avoiding double-negations is a bit clearer to understand.